wys/api/python/wys_api.py

```diff
@@ -470,19 +471,19 @@ def get_schedules(conn, api_key):
         response.raise_for_status()
         schedule_list = response.json()
     except RequestException as exc: 
-        logger.critical('Error querying API, %s', exc) 
-        sys.exit(2)
+        logger.critical('Error querying API, %s', exc)
+        logger.critical('Response: %s', exc.response)                        
 
     try:
         rows = [(schedule['name'], api_id)
                     for schedule in schedule_list if schedule['assigned_on_locations']
                         for api_id in schedule['assigned_on_locations'] if api_id]
     except TypeError as e:
         logger.critical('Error converting schedules response to values list.')
-        logger.critical('Return value: ', schedule_list)
+        logger.critical('Return value: %s', schedule_list)
         raise WYS_APIException(e)
 
-    with conn.cursor() as cur:
+    with conn.get_conn() as con, con.cursor() as cur:
```

Comment:
this connection opening should be moved back to the dag file but inside the decorated task function